app/src/main/java/com/example/notes2text/usecases/FileEditingUseCases/EditTextInteractor.java

```diff
@@ -0,0 +1,36 @@
+package com.example.notes2text.usecases.FileEditingUseCases;
+
+
+import android.text.Spannable;
+
+
+public class EditTextInteractor implements EditTextBoundary{
+
+    private String content;
+
+    public EditTextInteractor(Spannable text) {
+        this.content = text.toString();
+    }
+
+    // Capitalize text selected by user
+    public String capitalize(int start, int end) {
+        String unedited = this.content;
+        String textToChange = this.content.substring(start, end);
+        String edited = textToChange.toUpperCase();
+        String altered = unedited.replace(textToChange, edited);
+        this.content = altered;
+
+        return altered;
+    }
+
+    // Lowercase text selected by user
+    public String lowercase(int start, int end) {
+        String unedited = this.content;
+        String textToChange = this.content.substring(start, end);
+        String edited = textToChange.toLowerCase();
+        String altered = unedited.replace(textToChange, edited);
+        this.content = altered;
+
+        return altered;
+    }
```

Comment:
If there is time to add bolding and italicizing, you could wrap the selected text in markdown - which should let it display bolding - or create alternative, child classes to deal with if you have a doc file.